dev/com.ibm.ws.security.openidconnect.clients.common/resources/com/ibm/ws/security/openidconnect/clients/common/resources/OidcClientMessages.nlsprops

```diff
@@ -310,6 +307,11 @@ ERROR_SENDING_AUTHORIZATION_REQUEST=CWWKS1553E: The {0} OpenID Connect client en
 ERROR_SENDING_AUTHORIZATION_REQUEST.explanation=The OpenID Connect client configuration might be missing information, or the client encountered an error while communicating with the OpenID Connect provider.
 ERROR_SENDING_AUTHORIZATION_REQUEST.useraction=See the error in the message for more information.
 
+# do not translate hostname, wasReqURLRedirectDomainNames, and webAppSecurity
+MALFORMED_URL_IN_ORIGIN_REQUEST_URL_COOKIE=CWWKS1554E: A request to [{0}] is not valid. A required cookie with a name that begins with [{1}] has an incorrect value. The hostname [{2}] that is used in the cookie does not match the hostname of the request, and the hostname in the cookie is not one of the allowed domain names.
+MALFORMED_URL_IN_ORIGIN_REQUEST_URL_COOKIE.explanation=The request includes a malformed cookie. The hostname in the cookie must match the current request or must match one of the allowed domain names. The cookie might be modified from within the user agent.
```

Comment:
The message text is well written but "might be modified" is a bit confusing. Did you mean "was modified"? "might have been modified"? Are you suggesting that the user modify the cookie or look at the hostname in the cookie?